packages/journalforing/src/components/OppgaveTabellRad.tsx

```diff
@@ -5,7 +5,7 @@ import { NextFilled } from '@navikt/ds-icons';
 import { DateLabel } from '@navikt/ft-ui-komponenter';
 import OppgaveOversikt from '../typer/oppgaveOversiktTsType';
 import styles from './journalforingPanel.less';
-import { finnYtelseTekst } from './SakDetaljer';
+import { finnYtelseTekst } from './VelgSakForm';
```

Comment:
Mogleg du bør bruka Link (https://aksel.nav.no/komponenter/core/link) i staden for a-tag direkte lenger ned i denne komponenten